indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -37,4 +37,61 @@
 
         setupListGenerator(filterConfig);
     };
+
+    global.setupAbstractPage = function setupAbstractPage() {
+        $('body').on('click', '.judge-button', function(evt) {
+            var $box = $('#abstract-decision-box');
+            var $this = $(this);
+            var $form = $box.find('form');
+
+            evt.preventDefault();
+
+            $.ajax({
+                url: $this.data('href'),
+                method: 'POST',
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                data: getParamsFromSelectors($form.find('input:not(:disabled), textarea, select:not(:disabled)')),
+                success: function(data) {
+                    if (data.page_html) {
+                        $('.management-page').replaceWith(data.page_html);
+                        $box = $('#abstract-decision-box');
+                    } else {
+                        $box.replaceWith(data.box_html);
+                    }
+                    initForms($box);
```

Comment:
I think you need to call `showFormErrors` explicitly to get the error tooltips in case of server-side validation failure; `initForms` doesn't call it